contrib/local/configs/relayer-keys.json

```diff
@@ -0,0 +1,46 @@
+{
+  "chains": [
+    {
+      "name": "gaiad",
+      "id": "gaia",
+      "hdpath": "m/44'/118'/0'/0/0",
+      "keys": [
+        {
+          "name": "ryl-gaiad0",
```

Comment:
rename `ryl` to `rly` in entire file for key.name

Reply:
ahh thanks nice catch